flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
+}
```

Comment:
what makes it x86_64 only? We have a PyQt baseapp.

Reply:
The app uses different AI libraries to transcribe audio. The `req2flatpak` failed to generate dependency json if `--target-platforms` includes `aarch64`. I suspect if the dependencies are updated it will work on ARM as well, so ARM support may be future improvement. 

Hints on testing ARM apps would be very welcome. Any emulator? Or Raspberry PI?

Reply:
what dependency is it failing on? If it is PyQt6 the solution is to use the baseapp.

Reply:
`triton` that is used by `torch` has no ARM wheels. There is [open issue](https://github.com/triton-lang/triton/issues/5561) for this .

Reply:
the other thing is those nvidia cuda modules have no aarch64 packages either.